### ISSUE TITLE
OMNeT++ 6.0 compatibility

### ISSUE THREAD

**Issue by sk13** [riebl/artery#243]
I wonder if there is some interest / plans regarding OMNeT++ 6.0 compatibility.
I have some patches for making this happen, but I wonder if there is interest.
If yes, what is the minimal OMNeT++ version artery is expected to support?

**Comment by sk13** [comment]
Unfortunatelly OMNeT++ 6.0 compatibility does not imply INET 4.x -- since I am using INET 3.8.x (which is supported by OMNeT++ 6.0). As far as I understand moving to INET 4.x could mean a bigger effort (altough I have not tried it...). But maybe this could be a second step...

**Comment by riebl** [comment]
Yes, Artery shall run with OMNeT++ 6.0 in the foreseeable future. For some changes, I have been waiting for the actual OMNeT++ release because I don't want to "support" pre-releases.
The trouble is just that I have to do these things in my spare time; that's also the reason for my delayed response, sorry.

**Comment by wischhof** [comment]
Hi Raphael, just saw the discussion. FYI: We have ported most parts of Artery and Vanetza to INET 4.4 and OMNeT++ 6.0 as part of the CrowNet project, link to Artery fork is: https://github.com/roVer-HM/artery 
I think, you discussed a previous version with @stefanSchuhbaeck at the last OMNeT++ summit. Since then, we have ported more parts but we only test those parts of Artery which we use within CrowNet. So it is not finished but might be used as a basis. If it is useful for you, I'll send a pull request - did not do it until now, since it is only roughly tested and incomplete.

**Comment by riebl** [comment]
Thanks @wischhof for your hint/reminder! I will have a closer look at the forked repository as soon as I can spend some time for migration to OMNeT++ 6.0.
